rancher/v1.6/en/cattle/volumes/index.md

```diff
@@ -0,0 +1,222 @@
+
+When all containers attached to a volume are deleted, the volume enters `detached` state.
+
+A deleted volume is in `removed` state. These volumes no longer exist on any Docker host and won't appear in when listing volumes. The volume "shadow" objects exist in Rancher API for audit purposes.
+
+### Volume Scopes
+
+There are two different volume scopes. The scope refers to the level at which the volume is managed by Rancher.
+
+#### Environment Scope
+
+An `environment scope`, or `external` volume may be shared by all containers in an environment. Rancher schedules containers appropriately based on the volume driver's access mode. A user doesn't need to be aware of the technical
+
+The practical benefit of this scope is the ability to easily share data between different software services/stacks whose lifecycles are independently managed. The user is given full control and must [manage the volumes]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#managing-volumes) themselves.
+
+##### Shared Example
+
+Rancher requires environment volumes to already exist prior to creating a service consuming it. Any installed volume driver can be used. We will use the `local` driver because it ships with the Docker daemon.
+
+[Create the volume]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#create-a-volume).
+
+```
+$ rancher volume create --driver local redis-data-external
+```
+
+Define the volume with `external: true` modifier to indicate the volume's lifecycle is managed externally (environment scope). Save the `docker-compose.yml` file below to a directory.
+
+```yaml
+version: '2'
+services:
+  redis:
+    image: redis:3.0.7
+    volumes:
+    - redis-data-external:/data
+volumes:
+  redis-data-external:
+    driver: local
+    external: true
+```
+
+Now create the stack.
+
+```
+$ rancher stack create
+```
+
+[List the volumes]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#list-volumes) to verify that `redis-data` volume is `active`.
+
+
+> **Note:** Scaling a service up or down mounts or unmounts the same shared volume.
+
+##### Per-Container Example
+
+In environment scope, creating a unique volume for each container of a service is not supported. Consider using [Stack Scope]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#stack-scope{{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#list-volumes) volumes, or if absolutely necessary, [V1 Compose]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#v1-compose).
+
+#### Stack Scope
+
+A stack-scoped volume is owned and managed by the stack in which it is defined. The primary benefit is the volume lifecycle is automatically managed by Rancher as part of the stack lifecycle. Generic, self-documenting volume names may be defined without colliding with volume names in other stacks.
```

Comment:
The sentence "Generic, self-doc..." is good, but it kind of comes out of nowhere. Here is one possible way to clearify:
To avoid name collisions with other volumes, unique self-documenting names are used for the stack scope volumes you define. They include the stack name, original volume name, and a random component.